### PR TITLE
Add related collection to memos

### PR DESCRIPTION
**Pull request by clementbiron**
This changeset displays the related collection(s) for each memo and significantly improves the semantics and layout of the memos.

### CODE REVIEW

File: content/memos/linkedins-community-rules-are-not-translated-anymore.fr.md [hunk 1]
```diff
@@ -5,6 +5,7 @@ service: "LinkedIn"
 terms_types: ["Community Guidelines"]
 dates: ["2022-05-20"]
 author: Mathilde Saliou
+related_collections: ["france--elections"]
```
Comment (MattiSG) [inline content/memos/linkedins-community-rules-are-not-translated-anymore.fr.md:8]: This seems to be a typo.
```suggestion
related_collections: ["france-elections"]
```

File: content/memos/meta-adds-anti-suicide-action.fr.md [hunk 2]
```diff
@@ -2,9 +2,10 @@
 html_description: "Facebook a ajouté des 'ressources d'aide' pour les utilisateurs qui publient sur le suicide ou l'automutilation depuis le 13 juin, à la suite d'un examen antérieur depuis 2019."
 title: "Meta ajoute des actions de lutte contre le suicide"
 service: "Facebook"
-terms_types: ["Community Guidelines - Self-harm"]
+terms_types: ["Community Guidelines -- Self-harm"]
```
Comment (MattiSG) [inline content/memos/meta-adds-anti-suicide-action.fr.md:5]: Why this change? 🤔

File: themes/opentermsarchive/layouts/partials/memo.html [hunk 3]
```diff
@@ -0,0 +1,36 @@
+{{ $template := default "single" .template }}
+
+<article class="memo{{ with .class }} {{ . }}{{ end }}">
+  <header>
+    {{ if eq $template "list" }}<h4 class="memo__title"><a href="{{ .memo.RelPermalink }}">{{ .memo.Title }}</a></h4>{{ end}}
```
Comment (MattiSG) [inline themes/opentermsarchive/layouts/partials/memo.html:5]: Inconsistent whitespace in `{{ end}}`, plus all check failures outlined by GitHub Actions.

**Review by MattiSG — COMMENTED** [review]
I like the update to the list layout. However, I am not sure that listing the author in the list is very relevant, and it adds complexity to the layout (both in terms of amount of information and in number of header sizes). If anything, I would find it more relevant to add the source collection name!

I see that you considered that there might be several “related collections”. I had more in mind that there would be one source collection. Do we have examples of multiple collections being used? Do we foresee such cases?

Finally, I find the memo view itself a bit imbalanced, with the related collections taking a lot of space compared to the memo itself. But that might be too complicated to address.

Please request (and, in this case, re-request) a review only when all automated tests 😉

File: content/memos/meta-adds-anti-suicide-action.fr.md (hunk 2 above)
Comment (clementbiron) [reply content/memos/meta-adds-anti-suicide-action.fr.md:5]: It's a mistake.

**Comment by clementbiron** [comment]
As seen in synchro, I've removed the author display in the listing template and I agree with your other feedback, which could be taken into account later.

**Review by MattiSG — APPROVED** [review]
Looks good, thanks! 👍 

It would be good to have automated tests that check that the related collections exist in the list, and that there is at least one collection 🙂
